src/kaltura-live-plugin.tsx

```diff
@@ -101,6 +104,46 @@ export class KalturaLivePlugin extends KalturaPlayer.core.BasePlugin implements
     });
   }
 
+  private _makeBackgroundPlayers() {
+    const {preOfflineEntryId, postOfflineEntryId} = this.config;
+    this._preOfflinePlayer = preOfflineEntryId ? this._createBackgroundPlayer(preOfflineEntryId, 'pre-broadcast') : undefined;
+    this._postOfflinePlayer = postOfflineEntryId ? this._createBackgroundPlayer(postOfflineEntryId, 'post-broadcast') : undefined;
+  }
+
+  private _createBackgroundPlayer(entryId: string, tag: string) {
+    const targetId = `${tag}-${entryId}`;
+    let playerPlaceholder = document.createElement('div');
+    playerPlaceholder.setAttribute('id', targetId);
+    playerPlaceholder.hidden = true;
+    document.body.appendChild(playerPlaceholder);
+    const backgroundPlayerConfig = {
+      targetId,
+      disableUserCache: true,
+      playback: {
+        muted: true,
+        autoplay: false,
+        loop: true
+      },
+      ui: {
+        disable: true
+      },
+      provider: {
+        ...this.player.config.provider,
+        ignoreServerConfig: true
+      }
+    };
+    const backgroundPlayer = (window as any).KalturaPlayer.setup(backgroundPlayerConfig);
```

Comment:
- what if we already have a player with this targetId ? 
- what is this player needed for ?

Reply:
1. The question not clear for me, background players initialisation happens in constructor of live-plugin (only once), destroying of the players happens on `destroy` hook, targetID makes dynamically and it's hardly possible if someone use `pre-broadcast-<entry_id>` target.
2. background players needed to play kaltura-entry media as background for pre/post broadcast slate.